src/silx/gui/hdf5/Hdf5TreeModel.py

```diff
@@ -61,6 +62,8 @@ def _createRootLabel(h5obj):
         if path.startswith("/"):
             path = path[1:]
         label = "%s::%s" % (filename, path)
+        if isinstance(h5obj, DatasetSlice):
+            label += str(list(h5obj.indices))
```

Comment:
I'm not a fan of having specific code for `DatasetSlice` here, but I couldn't find a better way, except maybe adding this in the `silx.io.commonh5` API...